livekit-agents/livekit/agents/transcription/tts_forwarder.py

```diff
@@ -65,11 +68,13 @@ def __init__(
         participant: rtc.Participant | str,
         track: rtc.Track | rtc.TrackPublication | str | None = None,
         language: str = "",
-        speed: float = 4,
+        speed: float = 3.83,
```

Comment:
what does speed & new_sentence_delay indicate? some doc strings would be helpful here